types/ext/language-transformer.d.ts

```diff
@@ -31,6 +31,7 @@ export type ConditionMapEntries = ConditionMapEntry[];
 
 export type Condition = {
     name: string;
+    aliases?: string[];
```

Comment:
Can you describe a bit how this is used?

Reply:
Maybe I should just remove it, since i guess it might clutter the 'condition namespace' for a language; but what it does is lets e.g. noun be referred to as both 'noun' and 'n' in dictionaries and the transforms rules. I think it also fits what you mentioned about abbreviations in https://github.com/themoeway/yomitan/pull/687#issuecomment-1951373313. 

Though you'd probably want to use the shorter aliases in dictionaries and the longer in the transforms, what actually happened is I allowed this early on in Yezichak, and all the [KTY](https://github.com/themoeway/kaikki-to-yomitan) dicts use the longhand 'noun', 'verb' etc.  If we do remove it, I could fix the dicts in the next release, but a full converter run takes a while on my machine.

Reply:
I would suggest that this extension have a single canonical value, I find that the stricter an API or file format is with regards to what it accepts/expects, the easier it is to maintain. Which one is chosen probably doesn't make a huge difference overall, the abbreviation potentially has the benefit of smaller dictionary file size (although this is zipped+compressed), and also the IndexedDB storage size.

I was a bit unclear in what I was describing in that comment, but in my mind I was thinking of:
* Dictionaries can use abbreviations, since they have _lots_ of data
* Language transforms could potentially use the longer form

My rationale here was basically that dictionaries aren't hand-crafted, they are made by a script, so being able to directly read and quickly comprehend them isn't a huge issue. Compared with the language transform files, which _are_ hand-crafted and are already somewhat difficult to understand, so I _could_ see there being benefit to not abbreviating.